fe1-web/src/features/digital-cash/screens/SendReceive/SendReceive.tsx

```diff
@@ -300,6 +304,22 @@ export const SendReceiveHeaderRight = () => {
 
   const popToken = useMemo(() => rollCallToken?.token.publicKey.valueOf() || '', [rollCallToken]);
 
+  const serializedPopToken = useMemo(() => {
+    try {
+      return ScannablePopToken.encodePopToken({ pop_token: popToken });
+    } catch {
+      return null;
+    }
```

Comment:
Why don't you return an empty string there directly then (in catch)?
It seems that you return null to convert it in an empty string after if it's null.

Reply:
The idea was that if the popToken is null, it displays nothing (the return null after the toast show). In reality the `|| ''` condition will never be used (I have removed it now)